src/platform/packages/shared/response-ops/alerts-delete/components/rule_settings_section.tsx

```diff
@@ -11,11 +11,19 @@ import React, { lazy, Suspense, useState } from 'react';
 import { EuiButton, EuiDescribedFormGroup } from '@elastic/eui';
 import { EuiText } from '@elastic/eui';
 import { css } from '@emotion/react';
+import { HttpStart } from '@kbn/core/public';
 import * as i18n from '../translations';
 
 const ModalComponent = lazy(() => import('./modal'));
 
-export const AlertDeleteRuleSettingsSection = () => {
+interface AlertDeleteDescriptiveFormGroupProps {
+  services: { http: HttpStart };
+  isDisabled?: boolean;
+}
+export const AlertDeleteDescriptiveFormGroup = ({
```

Comment:
renaming to use the same name as EUI component used inside. I think it will be easier to understand how it looks like using this name